.ebextensions/cwl-logs-application.config

```diff
@@ -0,0 +1,40 @@
+# See cwl-logs-apache-access.config for more comments
+Mappings:
+  CWLogs:
+    ApplicationLogs:
+      LogDirectory: "/var/log/digitalmarketplace"
+      LogFile: "/var/log/digitalmarketplace/application.log"
+      TimestampFormat: "%Y-%m-%dT%H:%M:%S"
+      LogGroupName: {"Fn::GetOptionSetting": {"OptionName": "LogGroupName"}}
+      ApplicationName: {"Fn::GetOptionSetting": {"OptionName": "ApplicationName"}}
+
+Resources:
+  AWSEBAutoScalingGroup:
+    Metadata:
+      "AWS::CloudFormation::Init":
+        CWLogsAgentConfigSetup:
+          files:
+            "/tmp/cwlogs/conf.d/application.conf":
+              content : |
+                [application-logs]
+                file = `{"Fn::FindInMap":["CWLogs", "ApplicationLogs", "LogFile"]}`
+                log_group_name = `{"Fn::FindInMap":["CWLogs", "ApplicationLogs", "LogGroupName"]}`
+                log_stream_name = `{"Fn::Join": ["-", ["application", {"Fn::FindInMap":["CWLogs", "ApplicationLogs", "ApplicationName"]}]]}`
+                datetime_format = `{"Fn::FindInMap":["CWLogs", "ApplicationLogs", "TimestampFormat"]}`
+                [application-logs-combined]
+                file = `{"Fn::FindInMap":["CWLogs", "ApplicationLogs", "LogFile"]}`
+                log_group_name = `{"Fn::FindInMap":["CWLogs", "ApplicationLogs", "LogGroupName"]}`
+                log_stream_name = application-combined
+                datetime_format = `{"Fn::FindInMap":["CWLogs", "ApplicationLogs", "TimestampFormat"]}`
+              mode  : "000400"
+              owner : root
+              group : root
```

Comment:
Is 'root' the right owner and group for this?  Or would we be better with a 'logs' user/group?